crypto/stark-verifier-ethereum/test/primfield_test.ts

```diff
@@ -0,0 +1,43 @@
+import { deployContract, getWallets, solidity } from "ethereum-waffle";
+import { waffle } from "@nomiclabs/buidler";
+import chai from "chai";
+import {tx_to_events} from './test_utils';
+
+import PrimeFieldTesterArtifact from "../artifacts/PrimeFieldTester.json";
+import { PrimeFieldTester } from "../typechain/PrimeFieldTester";
+
+chai.use(solidity);
+const { expect } = chai;
+
+describe("Public coin testing", () => {
+  let field_contract: any;
+
+  const provider = waffle.provider;
+  let [wallet] = provider.getWallets();
+
+  before(async function() {
+    field_contract = await deployContract(wallet, PrimeFieldTesterArtifact) as PrimeFieldTester;
+  });
+
+  it("Sould add correctly", async function() {
```

Comment:
This is ok, but it would be better if it was a parameterized test comparing it to the native JS BigInt implementation.